crates/env/src/engine/on_chain/ext.rs

```diff
@@ -430,6 +430,27 @@ mod sys {
             out_len_ptr: Ptr32Mut<u32>,
         ) -> ReturnCode;
     }
+
+    #[link(wasm_import_module = "seal2")]
+    extern "C" {
+        // # Parameters
+        //
+        // - `key_ptr`: pointer into the linear memory where the location to store the value is placed.
+        // - `key_len`: the length of the key in bytes.
+        // - `value_ptr`: pointer into the linear memory where the value to set is placed.
+        // - `value_len`: the length of the value in bytes.
+        //
+        // # Return Value
+        //
+        // Returns the size of the pre-existing value at the specified key if any. Otherwise
+        // `SENTINEL` is returned as a sentinel value.
+        pub fn seal_set_storage(
```

Comment:
```suggestion
        pub fn set_storage(
```
I've just merged https://github.com/paritytech/ink/pull/1482, so `seal` prefix should be removed?